helpers/dmesg.py

```diff
@@ -48,16 +48,23 @@ def show(self):
         print(self.log)
 
     def _warn_count(self, warn_str):
-        match = re.findall(warn_str, self.log.decode())
-        return len(match)
+        return len(self._warn_match(warn_str))
 
-    def warn_count(self, warn_str):
+    def _warn_match(self, warn_str):
+        return re.findall(warn_str, self.log.decode())
+
+    def warn_count(self, warn_str: str) -> int:
```

Comment:
warn_str is already have expected variable type and its cool - so lets no complicate the code